ui/packages/shared/components/src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Button from './';
+
+export default {component: Button, title: 'Components/Button'};
+export const Primary = {args: {variant: 'primary', children: 'Primary', disabled: false}};
+export const Neutral = {args: {variant: 'neutral', children: 'Neutral', disabled: false}};
+export const Link = {args: {variant: 'link', children: 'Link Text', disabled: false}};
```

Comment:
is it possible to provide a user control in Storybook where we would be able to manually change the variant? For example a dropdown with all the available variant values?

Reply:
Yes, that's in the plan.
Right now, the controls for native prop types work out of the box. The boolean and string works, the variants is an enum, and it doesn't work out of the box, I'll see what can be done for that.

Reply:
Done, specifying the type explicitly worked here. Will proceed with that and then see if this can be inferred automatically some way
<img width="1648" alt="Screenshot 2023-03-16 at 4 43 31 PM" src="https://user-images.githubusercontent.com/1283424/225600530-1033b757-70fc-4a08-846f-85ffd07e282e.png">
.